Lib/uuid.py

```diff
@@ -663,6 +676,8 @@ def _random_getnode():
 _NODE_GETTERS_UNIX = [_unix_getnode, _ifconfig_getnode, _ip_getnode,
                       _arp_getnode, _lanscan_getnode, _netstat_getnode]
 
+_NODE_GETTERS_AIX = [_unix_getnode, _netstat_getnode]
```

Comment:
I would prefer to see a single _NODE_GETTERS constant and move the the platform test ("if sys.platform == 'win32':") here: move it from getnode().